src/helm/benchmark/test_model_properties.py

```diff
@@ -825,52 +825,62 @@
         ),
         max_sequence_length=511,
     ),
+    ModelDeployment(
+        name="google/gemini",
+        client_spec=ClientSpec(class_name="helm.proxy.clients.vertexai_client.VertexAIChatClient"),
+        tokenizer_name="hf-internal-testing/llama-tokenizer",
+        window_service_spec=WindowServiceSpec(
+            class_name="helm.benchmark.window_services.no_decoding_window_service.NoDecodingWindowService"
+        ),
+        max_sequence_length=30000,
```

Comment:
Should this be 30720 and 30720 + 2048? source: https://ai.google.dev/models/gemini#model_variations